src/NServiceBus.Testing.Fakes/TestableAuditContext.cs

```diff
@@ -2,37 +2,55 @@
 {
     using System;
     using System.Collections.Generic;
+    using NServiceBus.Audit;
     using Pipeline;
     using Transport;
 
     /// <summary>
     /// A testable implementation of <see cref="IAuditContext" />.
     /// </summary>
-    public partial class TestableAuditContext : TestableBehaviorContext, IAuditContext
+    public partial class TestableAuditContext : TestableBehaviorContext, IAuditContext, IAuditActionContext
     {
         /// <summary>
-        /// Contains the information added by <see cref="AddedAuditData" />.
+        /// Address of the audit queue.
         /// </summary>
-        public IDictionary<string, string> AddedAuditData { get; } = new Dictionary<string, string>();
+        public string AuditAddress { get; set; } = "audit queue address";
```

Comment:
It probably doesn't really matter, but I would use something without spaces to make it more like an actual address. Addresses would rarely ever contain spaces.

Reply:
fixed